### PR TITLE
New D3LineChart that draws a curved line

### PR DESCRIPTION
**Pull request by joehybird**
TODO : Add links to listview on line dots

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -47,7 +47,12 @@
                          - activities__set_activity_dates
                     - Display "busy" events (no collision) with a stripped background.
                     - Support timezone provided by the user information instead of browser.
-
+            * Reports:
+                - Add 'linechart' to the available charts.
```
Comment (genglert) [inline CHANGELOG.txt:51]: It's a user-sided feature.

File: creme/reports/locale/fr/LC_MESSAGES/django.po
Comment (genglert) [inline creme/reports/locale/fr/LC_MESSAGES/django.po:1]: Update the date in the header too.

File: creme/sketch/bricks.py [hunk 2]
```diff
@@ -192,6 +192,40 @@ def get_chart_props(self, context):
...
+    def get_chart_props(self, context):
+        props = {
+            "limits": self.limits,
+            "transition": self.enable_transition,
+            "xAxisTitle": self.abscissa_title,
+            "yAxisTitle": self.ordinate_title,
+            "showTooltip": self.enable_tooltip,
+        }
+
+        return props
```
Comment (genglert) [inline creme/sketch/bricks.py:226]: Useless variable.

**Review by genglert — CHANGES_REQUESTED** [review]
What if the user use JQplot? I presume it won't work => should we officially remove JQplot?

**Comment by joehybird** [comment]
> What if the user use JQplot? I presume it won't work => should we officially remove JQplot?

Yes it won't work. And yes we should officially remove jqplot, the only missing feature (for me) was the tooltip and it is done in this PR event if not used in the other charts, but it is easy to add now.

**Review by genglert — APPROVED** [review]

**Pull request merged**